.codecov.yaml

```diff
@@ -0,0 +1,14 @@
+coverage:
+  range: 80..100
+  round: down
+  precision: 2
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 # context, you can create multiple ones with custom titles
+        enabled: yes           # must be yes|true to enable this status
+        target: auto           # specify the target coverage for each commit status
+                               #   option: "auto" (must increase from parent commit or pull request base)
```

Comment:
```suggestion
                               #   option: "auto" (must not decrease from parent commit or pull request base)
```

Right?

Reply:
Right, actually I found out that we can do `auto` with a `threshold` https://docs.codecov.com/docs/commit-status and I have set this to be 5%, meaning that the coverage cannot drop more than 5% (to also allow some flakiness). 

Let me know if you think we should be stricter about this :)

Reply:
SGTM!